### PR TITLE
docs + fix

### PR DESCRIPTION
**Pull request by inkredabull**

### CODE REVIEW

**Comment by inkredabull** [comment]
### pre
Screen Shot 2020-12-11 at 12 32 39 PM

### post
Screen Shot 2020-12-11 at 12 47 48 PM

**Comment by coleby** [comment]
I'm confused by the screenshots

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Setup
+
+* `yarn install`
+* `yarn build`
```
Comment (coleby) [inline README.md:4]: `yarn start`?

Comment (sulf) [reply README.md:4]: `yarn start` terser should work fine with build.

**Comment by sulf** [comment]
oh but it doesnt!

**Review by sulf — APPROVED** [review]

**Comment by inkredabull** [comment]
pre : was broken / post : not broken anymore :-D

>

**Pull request merged**